tests/unit/utilities/addon-calculate-cache-key-for-tree-test.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var calculateCacheKeyForTree = require('../../../lib/utilities/addon-calculate-cache-key-for-tree');
+var chai = require('../../chai');
+var expect = chai.expect;
+
+describe('calculateCacheKeyForTree', function() {
+  it('should return same value for same input', function() {
+    var addon = {
+      root: 'hoy',
+      pkg: { some: 'parsed', value: { goes: 'here' }}
+    };
+
+    var first = calculateCacheKeyForTree('app', addon);
+    var second = calculateCacheKeyForTree('app', addon);
+
+    expect(first).to.equal(second);
+  });
+
+  it('should return different value when different', function() {
```

Comment:
This test doesn't seem to match this description. Also, I feel like the description should be something to the effect of `should return different values for different addon instances`?